lib/datasource/docker/index.ts

```diff
@@ -451,18 +451,46 @@ async function getTags(
     // AWS ECR limits the maximum number of results to 1000
     // See https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_DescribeRepositories.html#ECR-DescribeRepositories-request-maxResults
     const limit = ecrRegex.test(registry) ? 1000 : 10000;
-    let url = `${registry}/v2/${repository}/tags/list?n=${limit}`;
+
+    let url: string;
+    let page = 1;
+
+    // quay registry returns unordered tags using v2, let's use v1 for now
+    const isQuay = registry.endsWith('quay.io');
```

Comment:
@rarkins i think the quay specific code should be moved to separate file, like `./quay.ts`

Reply:
Or v1/v2 files?